ragna/deploy/_ui/central_view.py

```diff
@@ -158,27 +89,24 @@ def __init__(
             show_copy_icon=False,
             css_classes=[f"message-{role}"],
         )
-        self._stylesheets.extend(message_stylesheets)
+        self._stylesheets.append("css/chat_interface/chatmessage.css")
 
     def _copy_and_source_view_buttons(self) -> pn.Row:
+        source_info_button = pn.widgets.Button(
```

Comment:
Any reason why we need to define before we create the row? If not, can we move it back into the `pn.Row`?

Reply:
I changed this because that was more convenient while debugging, I needed to check properties on the button itself. I'm moving it back into `pn.Row`